homeworks/12/chat-system/src/Server.java

```diff
@@ -0,0 +1,33 @@
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class Server {
+    private ServerSocket serverSocket;
+
+    public Server(ServerSocket serverSocket) {
+        this.serverSocket = serverSocket;
+    }
+
+    public void startServer() {
+        try {
+            while (!serverSocket.isClosed()) {
+                Socket socket = serverSocket.accept();

```

Comment:
As long as it is a client connection and will later be passed to **ClientHandler**, I think that it would have been better if you had named this variable "_client_" instead of  "_socket_" 

`Socket client = serverSocket.accept();`